services/agents/registry.go

```diff
@@ -408,6 +408,17 @@ func (r *Registry) SendSetStateRequest(ctx context.Context, pmmAgentID string) {
 		return
 	}
 
+	pmmAgent, err := models.FindAgentByID(r.db.Querier, pmmAgentID)
+	if err != nil {
+		l.Errorf("Failed to get PMM Agent: %s.", err)
+		return
+	}
+	pmmAgentVersion, err := version.Parse(*pmmAgent.Version)
+	if err != nil {
+		l.Errorf("Failed to collect agents: %s.", err)
```

Comment:
Could you check when we added setting version into pmm-agent? If it was done before PMM-2.0 then it's ok.

Reply:
It was added in PMM-2.0. It doesn't exist in master branch. 
Shouldn't we check for errors anyway?

Reply:
I mean which version of PMM-2.0? before september of 2019 or after?